migrate.cfg.sagetracmigrationarchive

```diff
@@ -41,6 +47,43 @@ keywords_to_labels: {'beginner': 'good first issue'}
 # Migrate milestones
 migrate_milestones: yes
 
+# Map for certain Trac milestones to GitHub labels
+milestones_to_labels = {
+ 'sage-duplicate/invalid/wontfix': 'duplicate/invalid/wontfix',
+ 'sage-duplicate/invalid': 'duplicate/invalid/wontfix',
+ 'sage-duplicate': 'duplicate/invalid/wontfix',
+ 'sage-wait': 'pending',
+ 'sage-pending': 'pending',
+ 'sage-feature': 'feature',
+ 'sage-wishlist': 'wishlist',
+ 'sage-combinat': 'component: combinatorics',
+ 'sage-symbolics': 'component: symbolics',
+ 'sage-i18n': 'component: translations'}
+
+# Map for canceld Trac milestones to GitHub milestones
+canceled_milestones = {
+ 'sage-2.8.4.3': 'sage-2.8.5',
+ 'sage-3.2.4': 'sage-3.3',
+ 'sage-4.0.3': 'sage-4.1',
+ 'sage-4.1.3': 'sage-4.2',
+ 'sage-4.4.5': 'sage-4.5',
+ 'sage-4.7.3': 'sage-4.8',
+ 'sage-6.11': 'sage-7.0',
+ 'sage-7.7': 'sage-8.0'}
+
+# Map for certain Trac components to GitHub labels
+components_to_labels = {
+ 'solaris': 'porting: solaris',
+ 'cygwin': 'porting: cygwin',
+ 'freebsd': 'porting: bsd',
+ 'aix or hp-ux ports': 'porting: aix or hp-ux',
```

Comment:
Should these not be prefixed with "component: "?

Reply:
This happens in the `mapcomponent` function (as it has been before).

Reply:
Then perhaps the configuration variable should be renamed to `renamed_components` or something like this?

Reply:
Yes, now I see your point. But your first suggestion seems preferable to me since prefixing these labels is something project specific, too. Or we may introduce a separate variable `component_label_prefix` for this. But this would be less flexible compared to just moving the prefixes to the `components_to_labels` dictionary.

Reply:
I've made this change now in 9120e2a

Reply:
Thanks!